### DISCUSSION TITLE
Projects Beta - Reorder input fields on dropdown menu in sidebar of issue

### DISCUSSION THREAD

**Discussion by edblayney** [community/community#14074] [Projects and Issues] [answered]
When I add an issue to a Project (Beta) board inside the Issue description, I would like to be able to reorder the way the input fields appear.

Since we've been iterating through our sprint planning processes, we've deleted and added fields. And, new fields are always appended to the bottom of the sidebar dropdown menu list no matter their importance.

I would like to be able to reorder them in order of importance or frequency of use to help with my own data tracking and limit barriers for my devs & product partners to add the data our product team needs to be successful in our planning efforts.

**Comment by jthurne** [comment, marked as answer]
Looks like a potential duplicate of https://github.com/github/feedback/discussions/10044

**Reply by edblayney** [reply]
It is...thank you
